app/screens/Login.java

```diff
@@ -6,29 +6,35 @@
 import icebase.app.enums.SCREEN_ENUM;
 import icebase.icebase.Auth;
 import icebase.icebase.User;
+import icebase.icebase.exceptions.UserNotFoundException;
 
 public class Login implements Screen {
 
     public void display() {
         Scanner sc = App.sc;
+        Auth auth;
+        User user;
 
         // Need while loop for error messages
         while (true) {
             MenuTitle.displayMainTitle();
-            MenuTitle.displaySubTitle("[Log-in]");
+            MenuTitle.displaySubTitle("Log-in");
             try {
-                Auth auth = Auth.getAuth();
+                auth = Auth.getAuth();
 
                 System.out.print("Username: ");
                 String username = sc.nextLine();
                 System.out.print("Password: ");
                 String password = sc.nextLine();
 
-                User user = auth.login(username, password);
+                user = auth.login(username, password);
 
                 if (user != null) {
                     break;
                 }
+            } catch (UserNotFoundException e) {
+                System.out.println(e.getMessage());
+                return;
```

Comment:
why mag return if user not found?

Reply:
para sana makasign-up muna yung user kay baka di niya intention na mag-log-in ganyan. pwede rin di siya magreturn kay baka natypo lang. magcreate na lang ba ko way para mahandle yung cases na yun? or iassume ko lang gatypo siya everytime so di na magreturn

Reply:
ah okay gets, so meaning if wala siya alam nga user ma stuck siya forever sa while loop


gets gets ok will resolve